near/devnet_deploy.ts

```diff
@@ -0,0 +1,322 @@
+// npx pretty-quick
+
+const nearAPI = require("near-api-js");
+const BN = require("bn.js");
+const fs = require("fs");
+const fetch = require("node-fetch");
+import { NodeHttpTransport } from "@improbable-eng/grpc-web-node-http-transport";
+
+function getConfig(env: any) {
+  switch (env) {
+    case "sandbox":
+    case "local":
+      return {
+        networkId: "sandbox",
+        nodeUrl: "http://localhost:3030",
+        masterAccount: "test.near",
+        wormholeAccount: "wormhole.test.near",
+        tokenAccount: "token.test.near",
+        nftAccount: "nft.test.near",
+        testAccount: "test.test.near",
+      };
+    case "testnet":
+      return {
+        networkId: "testnet",
+        nodeUrl: "https://rpc.testnet.near.org",
+        masterAccount: "wormhole.testnet",
+        wormholeAccount: "wormhole.wormhole.testnet",
+        tokenAccount: "token.wormhole.testnet",
+        nftAccount: "nft.wormhole.testnet",
+        testAccount: "test.wormhole.testnet",
+      };
+  }
+  return {};
+}
+
+async function initNear() {
+  let e = process.env.NEAR_ENV || "sandbox";
+
+  let config = getConfig(e);
+
+  let masterKey: any;
+
+  if (e == "sandbox") {
```

Comment:
General comment for file.  Maybe use "==="?

Reply:
fixed